plutus-core/changelog.d/20230203_104716_thealmartyblog_fix_goldenPIR_parser_err_show.md

```diff
@@ -0,0 +1,41 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Removed
+
+- A bullet item for the Removed category.
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category.
+
+-->
+<!--
+### Changed
+
+- A bullet item for the Changed category.
+
+-->
+<!--
+### Deprecated
+
+- A bullet item for the Deprecated category.
+
+-->
+
+### Fixed
+
+- Printing of parser errors of the `goldenPIR` function.
```

Comment:
This is okay, but it could be more informative. I'm not sure what the reader of the changelog is going to take away from this beyond "bug fixed". Could say "The `goldenPIR` function now uses the correct function to print parse errors, so they are now printed in a human-readable way."